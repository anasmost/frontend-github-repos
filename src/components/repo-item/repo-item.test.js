import { shallow } from "enzyme";
import RepoItem from "./repo-item";

import React from "react";

describe("<RepoItem/>", () => {
  it("renders shallowly RepoItem", () => {
    const mockRepos = {
      owner: {},
    };
    expect.assertions(1);
    expect(shallow(<RepoItem {...mockRepos} />)).toMatchSnapshot();
  });
});
