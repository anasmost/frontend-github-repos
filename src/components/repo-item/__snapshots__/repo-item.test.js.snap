// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RepoItem/> renders shallowly RepoItem 1`] = `ShallowWrapper {}`;
